changelog.d/5534.bugfix

```diff
@@ -0,0 +1 @@
+Fixes crash upon opening timeline due to additionalTopSpace
```

Comment:
I am wondering if this fix needs an entry. Is the crash on the current release?

Reply:
After several discussions, updated the PR and removed the changelog 👍️